core/src/dsnp/pseudo_relationship_identifier.rs

```diff
@@ -0,0 +1,91 @@
+#![allow(dead_code)] // todo: remove after usage
+use crate::dsnp::dsnp_types::{DsnpId, DsnpPrid};
+use anyhow::{Error, Result};
+use dryoc::{
+	classic::{
+		crypto_box::crypto_box_beforenm,
+		crypto_secretbox::{crypto_secretbox_detached, Nonce},
+	},
+	constants::CRYPTO_SECRETBOX_MACBYTES,
+	kdf::{Key, StackKdf},
+	keypair::{PublicKey, SecretKey},
+	types::ByteArray,
+};
+use std::ops::Deref;
+use zeroize::Zeroizing;
+
+const PRI_CONTEXT: &[u8] = b"PRIdCtx0";
+struct PseudonymousRelationshipIdentifier;
+
+impl PseudonymousRelationshipIdentifier {
```

Comment:
Should we merge this struct with `DsnpPrid` already defined?

Reply:
Sounds good.